pgptool-gui/installer/launch4j.xml

```diff
@@ -2,8 +2,8 @@
 <launch4jConfig>
   <dontWrapJar>true</dontWrapJar>
   <headerType>gui</headerType>
-  <jar>pgptoolgui.jar</jar>
-  <outfile>/home/sergeyk/_prj/pgptool/pgptool-gui/target/pgptool.exe</outfile>
+  <jar>${project.build.directory}/pgptoolgui-${version}.jar</jar>
```

Comment:
@ctrlcctrlv can you please share -- why you've added version here? Is it matter of "taste" or is it required for something to work?

Reply:
Required to work on my system.